### DISCUSSION TITLE
what energy dpdata reads in the VASP OUTCAR file

### DISCUSSION THREAD

**Discussion by chandagsin** [deepmodeling/dpdata#411] [Ideas]
Dear Developers,

I practised with a dataset downloaded from the official website,
https://tutorials.deepmodeling.com/en/latest/Tutorials/DeePMD-kit/learnDoc/Handson-Tutorial%28v2.0.3%29.html

$ wget https://dp-public.oss-cn-beijing.aliyuncs.com/community/CH4.tar

This is a VASP OUTCAR file.  The energy of each frame is given in this format (e.g.), 

  alpha Z        PSCENC =         0.24272866
  Ewald energy   TEWEN  =       126.94348121
  -Hartree energ DENC   =      -282.92879547
  -exchange      EXHF   =         0.00000000
  -V(xc)+E(xc)   XCENC  =        24.66067699
  PAW double counting   =       456.76874710     -457.55018458
  entropy T*S    EENTRO =         0.01836410
  eigenvalues    EBANDS =       -89.19239622
  atomic energy  EATOM  =       197.04175453
  Solvation  Ediel_sol  =         0.00000000

  free energy    TOTEN  =       -23.99562368 eV

  energy without entropy =      -24.01398778  energy(sigma->0) =      -24.00174505

I found that dpdata reads the "TOTEN" energy, which includes the entropy term "entropy T*S EENTRO". From what I understand, AIMD does not need this term to solve the equations of molecular motions. (although sometimes this term is indeed zero) Not sure this would be a bug; just let you know. 

Thanks,

chandagsin

**Comment by njzjz** [comment]
`energy without entropy` is not consistent with forces. You can see [the previous discussion in ASE](https://gitlab.com/ase/ase/-/issues/430). To train a machine learning model, we hope that energies are consistent with forces.
